src/measure_theory/integral/interval_integral.lean

```diff
@@ -802,10 +802,27 @@ begin
   { simp [hab, hab.le] }
 end
 
+lemma integral_Ioc_eq_integral_Ioo' {f : α → E} {a b : α} (hb : μ {b} = 0) :
+  ∫ t in Ioc a b, f t ∂μ = ∫ t in Ioo a b, f t ∂μ :=
```

Comment:
Could you please add a version of https://leanprover-community.github.io/mathlib_docs/measure_theory/measure/measure_space.html#measure_theory.Ioo_ae_eq_Ioc that assumes `μ {b} = 0`? Then you can just use https://leanprover-community.github.io/mathlib_docs/measure_theory/integral/set_integral.html#measure_theory.set_integral_congr_set_ae here.

Reply:
Done, and refactored the previous proof to do this too! Thanks for the idea!